core/mondoo-okta-security.mql.yaml

```diff
@@ -1,379 +1,1985 @@
+    title: Enable password changed notification for end users 
+    impact: 80
+    filters: asset.platform == "terraform-plan" && terraform.plan.resourceChanges.contains( providerName == /okta/ )
+    mql: terraform.plan.resourceChanges.where( type == /okta_security_notification_emails/ ).all( change.after['send_email_for_factor_reset_enabled'] == true )
+    docs: 
+      remediation: 
+        - id: terraform 
+          desc: |
+            ```
+            resource "okta_security_notification_emails" "example" {
+              report_suspicious_activity_enabled       = true
+              send_email_for_factor_enrollment_enabled = true
+              send_email_for_factor_reset_enabled      = true
+              send_email_for_new_device_enabled        = true
+              send_email_for_password_changed_enabled  = true
+            }
+            ```
+            
+  - uid: mondoo-okta-security-password-settings-minimum-length
+    title: Enable strong password settings for password policies - minimum length
+    impact: 30
+    props:
+      - uid: oktaPasswordLength
+        mql: 15
+    docs:
+      desc: |
+        Specify a minimum password length of at least eight characters. Longer passwords provide greater protection against brute force attacks.
+
+        Longer passwords are more difficult for users to remember, especially when combined with other complexity requirements (for example, require uppercase, lowercase, symbols, and so on). NIST recommends longer passwords that are easy to remember (“phrase-like”) but more difficult to obtain from brute force attacks.
+
+        These recommendations are provided by Okta as an example of typical password standards. They're derived from current cybersecurity industry best practices. They aren't intended to replace internationally recognized cybersecurity standards, such as ISO 27001, National Institute of Standards and Technology (NIST), PCI-DSS, or others. Your IT department may need to adjust these settings to comply with whichever cybersecurity standard your organization has chosen to follow.
+    variants:
+      - uid: mondoo-okta-security-password-settings-minimum-length-runtime
+      - uid: mondoo-okta-security-password-settings-minimum-length-terraform-hcl
+      - uid: mondoo-okta-security-password-settings-minimum-length-terraform-plan
+      - uid: mondoo-okta-security-password-settings-minimum-length-terraform-state
+  - uid: mondoo-okta-security-password-settings-minimum-length-runtime
+    title: Enable strong password settings for password policies - minimum length
+    impact: 30
+    filters: asset.platform == "okta-org"
+    docs:
+      remediation:
+        - id: console
+          desc: |
+            Configure password settings for password policies
+
+            1. In the Admin Console menu, go to Security > Authentication.
+            2. In the Password tab, review each policy. To edit the policy, click Edit.
```

Comment:
```suggestion
            2. In the Password tab, review each policy. To edit the policy, select Edit.
```

Reply:
@tas50 these are not our steps. they are from Okta's docs. They are copy/pasted.